site/content/docs/4.3/components/icons.md

```diff
@@ -11,21 +11,16 @@ toc: true
 Meet the Bootstrap icons, a limited set of icons designed first and foremost to work with Bootstrap's components. From form controls to navigation, these icons are the perfect complement to our toolkit. They're SVGs, so they'll scale quickly and easily to any size, they can be easily recolored with CSS, and more. They're also included in Bootstrap under the MIT license.
 
 <div class="bd-bi-grid d-flex flex-wrap mt-4 mb-5 mx-n2">
-  {{< listIcons.inline >}}
-  {{- $dirName := printf "site/static/docs/%s/dist/icons/" .Site.Params.docs_version -}}
-
-  {{- range (readDir $dirName) -}}
```

Comment:
@mdo: is the only reason for this patch the fact that you want the Bootstrap icons shown first? This adds an extra step which we could avoid. I don't mind either way, just thinking out loud.

I could probably hook something up to change the order, but it might complicate things.